basics/led_blink/README.md

```diff
@@ -0,0 +1,38 @@
+# Simple LED Blink
+
+## Introduction
+
+The LED Blink program is equivalent to the traditional "Hello World!" program that most coders begin their jouney with.
+
+In this program, we will make one of the onboard LEDs blink. You can choose either LED 1 or LED 2. It doesn't really make a difference. 
+Note that the PIN numbers for the LEDs are as follows:
+- LED1 - GPIO pin 33 
+- LED2 - GPIO pin 4
+
+Here, GPIO stands for "General Purpose Input Output". This means that the pin can be used for either output or input. We will have to specify this in our program.
+
+## Equipments Required
+
+| Name  | Specification | Quantity |
+| -- | -- | -- |
+| ASC CSE IoT 2.0 Board | - | 1 |
+| USB Type C Cable | Data cable | 1 |
+| Personal Computer | Arduino IDE Installed | 1 |
+
+## Connections
+
+No external connections to the board are required for this program. Just connect your board to your PC. Refer to [this](../README.md) if you are facing difficulties.
+
+## Code Explanation 
+
+<!-- To be added.  -->
+
+## Conclusion
+
+Yay! You've written your first program for embedded systems! If you're up for a challenge, try solving the problrms mentioned in **Extras**. When you're ready, head over to the [README](../button/README.md) for the onboard button to get started with handling inputs.
```

Comment:
Typographical error "problrms" --> "problems". 

It would be a better option to write as follows -
```
If you are up for a challenge, try implementing some of the tasks mentioned under the Extras section.
```